features/lorawan/LoRaWANStack.cpp

```diff
@@ -41,6 +41,16 @@ SPDX-License-Identifier: BSD-3-Clause
 
 #define INVALID_PORT                0xFF
 #define MAX_CONFIRMED_MSG_RETRIES   255
+/**
+ * Control flags for transient states
+ */
+#define IDLE_FLAG                   0x00000000
+#define TX_ONGOING_FLAG             0x00000001
+#define MSG_RECVD_FLAG              0x00000002
+#define BUSY_FLAG                   0x00000004
```

Comment:
BUSY_FLAG bit is only set and cleared but never checked so I thinki this should be removed.